### DISCUSSION TITLE
katana opening issue in kali after installing, command not found option found

### DISCUSSION THREAD

**Discussion by BlueHatOrganization** [projectdiscovery/katana#756] [Q&A] [answered]
### Describe the use case of this feature:

```[tasklist]
### Tasks
```

**Comment by olearycrew** [comment, marked as answer]
You most likely need to check your PATH as described here: https://docs.projectdiscovery.io/tools/katana/install

**Comment by Parveen-Birthaliya** [comment]
I don't have /.bashrc what should I do

**Comment by GeorginaReeder** [comment]
Hey @Parveen-Birthaliya - the /.bashrc file should be in your home directory. Are you seeing a 'not found' error when you try to add the PATH in your terminal, or something else?
